assets/airports/kchs.json

```diff
@@ -0,0 +1,10037 @@
+        "RSRVE": ["N33d12m34.546", "W79d40m23.133"],
+        "SAV":   ["N32d08m46.616", "W81d11m56.676"],
+        "SEAMS": ["N33d08m10.684", "W80d23m17.406"],
+        "SHRMP": ["N32d42m43.217", "W79d53m46.039"],
+        "SNOBB": ["N32d53m45.127", "W79d53m37.381"],
+        "SOGTY": ["N32d51m00.023", "W80d00m01.115"],
+        "STNNS": ["N32d49m29.521", "W79d58m08.461"],
+        "STUNO": ["N32d58m31.516", "W80d53m36.112"],
+        "SWPFX": ["N33d22m59.302", "W79d53m48.569"],
+        "TAWKE": ["N31d15m56.877", "W81d12m51.189"],
+        "TRTLS": ["N33d13m58.362", "W80d40m24.218"],
+        "TYLOR": ["N32d52m14.612", "W80d11m27.582"],
+        "UNJAM": ["N34d12m56.552", "W80d48m10.587"],
+        "UZADY": ["N32d46m18.656", "W80d02m47.093"],
+        "VAN":   ["N33d28m29.355", "W80d26m54.449", "Vance"],
+        "WANDO": ["N33d02m03.469", "W79d50m26.905"],
+        "WBODY": ["N34d08m54.495", "W81d37m54.319"],
+        "WILIT": ["N33d29m30.797", "W79d27m38.471"],
+        "WOGIB": ["N33d37m59.033", "W82d18m43.983"],
+        "ZUNSU": ["N32d49m33.581", "W80d04m36.975"]
+    },
+    "runways": [
+        {
+            "name": ["3", "21"],
+            "end": [
+                ["N32d53m05.173", "W80d02m43.914", "45ft"],
+                ["N32d54m08.270", "W80d02m10.208", "45ft"]
+            ],
+            "ils": [false, false]
+        },
+        {
+            "name": ["15", "33"],
+            "end": [
+                ["N32d54m46.761", "W80d02m53.168", "45ft"],
+                ["N32d53m31.386", "W80d01m56.156", "45ft"]
+            ],
+            "ils": [true, true]
+        }
+    ],
+    "sids": {
+        "LGRHD3": {
+            "icao": "LGRHD3",
+            "name": "Logger head Three",
+            "altitude": 4000,
+            "rwy": {
+                "KCHS3": ["HALEE", "LGRHD", "GIPPL"],
+                "KCHS15": ["LGRHD", "GIPPL"],
+                "KCHS21": ["LGRHD", "GIPPL"],
+                "KCHS33": ["HALEE", "LGRHD", "GIPPL"]
+            },
+            "exitPoints": {
+                "GIPPL": ["GIPPL"]
+            },
+            "draw": [
+                ["HALEE", "LGRHD", "GIPPL"],
+                ["LGRHD", "GIPPL"]
+            ]
+        },
+        "MLTRE2": {
+            "icao": "MLTRE2",
+            "name": "Molter Two",
+            "altitude": 4000,
+            "rwy": {
+                "KCHS3": ["MLTRE"],
+                "KCHS15": ["ANGRE", "MLTRE"],
+                "KCHS21": ["ANGRE", "MLTRE"],
+                "KCHS33": ["MLTRE"]
+            },
+            "exitPoints": {
+                "JYMSE": ["JYMSE"],
+                "VAN":   ["VAN"]
+            },
+            "draw": [
+                ["ANGRE", "MLTRE"],
+                ["MLTRE", "JYMSE*"],
+                ["MLTRE", "VAN*"]
+            ]
+        },
+        "PLMTO3": {
+            "icao": "PLMTO3",
+            "name": "Plumtree Three",
+            "altitude": 4000,
+            "rwy": {
```

Comment:
LOIRN is missing from 15 and 21. Ideally, LOIRN and PLMTO should be moved into the body.